frontend/src/js/controllers/featuredChallengeCtrl.js

```diff
@@ -130,6 +163,8 @@
                     var details = response.data;
                     vm.leaderboard = details.results;
                     for (var i=0; i<vm.leaderboard.length; i++) {
+                        vm.leaderboard[i]['submission__submitted_at_formatted'] = vm.leaderboard[i]['submission__submitted_at'];
+                        vm.initial_ranking[vm.leaderboard[i].submission__participant_team__team_name] = i+1;
```

Comment:
Follow the same behaviour of storing initial_ranking by team id to avoid confusions.

Reply:
Okay. Thanks for your review